src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx

```diff
@@ -144,23 +156,24 @@ export default class SecurityUserSettingsTab extends React.Component {
         }
     };
 
-    _getInvitedRooms = () => {
+    private getInvitedRooms = (): Room[] => {
         return MatrixClientPeg.get().getRooms().filter((r) => {
             return r.hasMembershipState(MatrixClientPeg.get().getUserId(), "invite");
         });
     };
 
-    _manageInvites = async (accept) => {
+    private manageInvites = async (accept: boolean): Promise<void> => {
         this.setState({
             managingInvites: true,
         });
 
         // Compile array of invitation room ids
-        const invitedRoomIds = this._getInvitedRooms().map((room) => {
+        const invitedRoomIds = this.getInvitedRooms().map((room) => {
             return room.roomId;
         });
 
         // Execute all acceptances/rejections sequentially
+        // eslint-disable-next-line @typescript-eslint/no-this-alias
         const self = this;
```

Comment:
I think we can just convert `self` to `this` in the one usage, given it's all arrow functions now.